docs/FAQ.md

```diff
@@ -99,15 +95,7 @@ plugins. ​
 
 ### What is a "plugin" in Backstage?
 
-​ Plugins are what provide the feature functionality in Backstage. They are used
-to integrate different systems into Backstage's frontend, so that the developer
-gets a consistent UX, no matter what tool or service is being accessed on the
-other side. ​ Each plugin is treated as a self-contained web app and can include
-almost any type of content. Plugins all use a common set of platform APIs and
-reusable UI components. Plugins can fetch data either from the backend or an API
-exposed through the proxy. ​ Learn more about
-[the different components](https://github.com/spotify/backstage#overview) that
-make up Backstage. ​
+By far, our most-used plugin is our TechDocs plugin, which we use for creating technical documentation. Our philosophy at Spotify is to treat "docs like code", where you write documentation using the same workflow as you write your code. This makes it easier to create, find, and update documentation. We hope to release [the open source version](https://github.com/spotify/backstage/issues/687) in the future. (See also: "[Will Spotify's internal plugins be open sourced, too?](https://github.com/spotify/faq#will-spotifys-internal-plugins-be-open-sourced-too)" above)
```

Comment:
Same issue as above.

Reply:
@nikkibeesetti @stefanalund This is the correct link for this:

https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage

Reply:
@nikkibeesetti sorry that was my bad, this link should be: 

https://github.com/spotify/backstage/blob/master/docs/FAQ.md#will-spotifys-internal-plugins-be-open-sourced-too

Reply:
You can use just `/docs/FAQ.md#will-spotifys-internal-plugins-be-open-sourced-too` as well